src/storage/structs.cairo

```diff
@@ -28,10 +71,9 @@ pub struct Execution {
     pub selector: felt252,
     /// The calldata to pass to entry point.
     pub calldata: StorageArrayFelt252,
-    /// The bool that indicates if execution is approved or not.
-    pub approved: bool,
-    /// The bool that indicates if execution is already executed or not.
-    pub executed: bool,
+    /// Bitmap that holds execution request status. index 0 is approved, index 1 is rejected, index
+    /// 2 is executed.
+    pub execution_request_status: u128,
 }
```

Comment:
Is a bitmap really necessary here if we already know all possible values and only 3?
Seems like a simple enum would be more straightforward, i.e.:
```cairo
self
    .Identity_executions
    .entry(execution_id)
    .execution_request_status
    .write(ExecutionStatus::Executed);
```